quil-rs/src/program/calibration.rs

```diff
@@ -398,6 +416,8 @@ mod tests {
                     "    PRAGMA INCORRECT_ORDERING\n",
                     "DEFCAL MEASURE 0 addr:\n",
                     "    PRAGMA CORRECT\n",
+                    "DEFCAL MEASURE q addr:\n",
+                    "    PRAGMA CORRECT\n",
```

Comment:
@kalzoo @notmgsk This is syntax `pyQuil` supports. A variable qubit, followed by an address. `quil-rs` bombs on this, because it only expects an optional fixed Qubit. So in this case it thinks the qubit was left out, reads `q` as the target, and then chokes on `addr`. Is this valid? If so, I have an issue and (potential) fix queued up.

Reply:
I have this "fixed" on the calibration matching side, but the parser needs to be updated as well. Here's the fix, targeting `main` since it would be beneficial there as well: https://github.com/rigetti/quil-rs/pull/159

Reply:
#159 was merged and propagated back up to this PR.